docs/en/mutations.md

```diff
@@ -67,6 +67,24 @@ mutations: {
 }
 ```
 
+### Quite Dispatch
```

Comment:
btw, maybe rename it into something like `silent`? because even English man did a mistake :D